lib/resources/windows_task.rb

```diff
@@ -4,7 +4,7 @@ class WindowsTasks < Inspec.resource(1)
     name 'windows_task'
     supports platform: 'windows'
     desc 'Use the windows_task InSpec audit resource to test task schedules on Microsoft Windows.'
-    example <<~EXAMPLE
```

Comment:
We have standardized on `<<~` because it fixes indentation errors in other documentation formats.  Please revert this change.

Reply:
Hi!
I'd need some help. Found windows_task_test.rb for unit tests and reviewed helper file however don't know how windows task is mocked and what values should I expect.

<<~EXAMPLE is reverted.

Reply:
Never mind. Done :)